### PR TITLE
chore: add ci to publish npm package

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -38,6 +37,16 @@ defaults_license_scanner: &defaults_license_scanner
...
+  name: Update NPM registry auth token
+  command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
+
+defaults_npm_publish_release: &defaults_npm_publish_release
+  name: Publish NPM $RELEASE_TAG artifact
+  command: |
+    source $BASH_ENV
+    echo "Publishing tag $RELEASE_TAG"
+    npm publish --tag $RELEASE_TAG --access public
 defaults_export_version_from_package: &defaults_export_version_from_package
```
Comment (mdebarros) [inline .circleci/config.yml:50]:
```suggestion

defaults_export_version_from_package: &defaults_export_version_from_package
```

**Review by mdebarros — APPROVED** [review]
Looks good. 

Made a small suggestion by adding a new line between defaults.

**Review by mdebarros — CHANGES_REQUESTED** [review]
Please see this PR: https://github.com/mojaloop/sdk-scheme-adapter/pull/332

I think we should similarly add an npmignore for the test folder here.

Let me know your thoughts.

**Comment by kleyow** [comment]
re: npmignore :+1:

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Ignore tests for publishing
+test/*
```
Comment (mdebarros) [inline .npmignore:2]:
```suggestion
test/*
.circleci/*
```

**Review by mdebarros — CHANGES_REQUESTED** [review]
+1

File: .npmignore (hunk 2 above)
Comment (kleyow) [reply .npmignore:2]: done

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**